pytest-selenium/pages/accounts.py

```diff
@@ -147,6 +149,8 @@ def _use_new_accounts_flow(self) -> bool:
         :rtype: bool
 
         """
+        WebDriverWait(self.driver, 1).until(
```

Comment:
PyPOM Pages should have a WDW already limiting the need for a new instance as `self.wait.until()`

Reply:
Obviously this wouldn't apply if you're looking for a specific limit on the wait time.